cluster/images/etcd-version-monitor/README.md

```diff
@@ -0,0 +1,30 @@
+# etcd-version-monitor
+
+This is a tool for exporting metrics exposed by etcd endpoints that are not
+in prometheus metrics format. The exported metrics would be in prometheus
+metrics format and would be exposed by this tool at localhost:9101/metrics
+http endpoint.
+
+**RUNNING THE TOOL**
+
+To run this tool as a binary:
+- make
+- ./bin/etcd-version-monitor
+
+To run this tool as a docker container:
+- make build-image
+- docker run --net=host -i -t gcr.io/google_containers/etcd_version_monitor:test /bin/etcd-version-monitor
+<br /> Optionally, change the 'IMAGE' variable inside the Makefile and run 'make push_image'
```

Comment:
Removed the point about building the image with a different name. However, wrote the note about running directly using docker for people who might want to use this but not in the context of k8s.
Do you suggest removing this point?